external/concealer/include/concealer/field_operator_application_for_autoware_universe.hpp

```diff
@@ -127,6 +129,7 @@ class FieldOperatorApplicationFor<AutowareUniverse>
     requestInitialPose("/api/localization/initialize", *this),
     // NOTE: /api/routing/set_route_points takes a long time to return. But the specified duration is not decided by any technical reasons.
     requestSetRoutePoints("/api/routing/set_route_points", *this, std::chrono::seconds(10)),
+    requestClearRoute("/api/routing/clear_route", *this),
```

Comment:
Please sort by lexicographical order.

Reply:
Thank you for this, I had overlooked it...